ee/clickhouse/queries/cohort_query.py

```diff
@@ -187,14 +188,22 @@ def __init__(
 
     @staticmethod
     def unwrap_cohort(filter: Filter, team_id: int) -> Filter:
-        def _unwrap(property_group: PropertyGroup) -> PropertyGroup:
+        def _unwrap(property_group: PropertyGroup, negate_group: bool = False) -> PropertyGroup:
             if len(property_group.values):
                 if isinstance(property_group.values[0], PropertyGroup):
                     # dealing with a list of property groups, so unwrap each one
-                    return PropertyGroup(
-                        type=property_group.type,
-                        values=[_unwrap(v) for v in cast(List[PropertyGroup], property_group.values)],
-                    )
+                    if not negate_group:
```

Comment:
would add a comment here that we're demorgan law'ing this so it makes sense when someone is reading this